keyboards/kiwikeebs/macro-v2/readme.md

```diff
@@ -0,0 +1,19 @@
+# KiwiKeebs MacroBoard V1.0
+
+![KiwiKeebs](https://i.imgur.com/h5i9jBl.png)
```

Comment:
This image is not valid, "has been removed".

Reply:
Added with updated static logo on website